cmd/testperformance.go

```diff
@@ -28,19 +29,42 @@ import (
 
 type testPerformanceConfig struct {
 	testConfig
-	DiskWriteMB                int
+	DiskIOTestFileDir          string
+	DiskIOBlockSizeKb          int
 	InternetTestServersOnly    []string
 	InternetTestServersExclude []string
 }
 
+type fioResult struct {
+	Jobs []fioResultJobs `json:"jobs"`
+}
+
+type fioResultJobs struct {
+	Read  fioResultSingle `json:"read"`
+	Write fioResultSingle `json:"write"`
+}
+
+type fioResultSingle struct {
+	Iops float64 `json:"iops"`
+	Bw   float64 `json:"bw"`
+}
+
 const (
-	diskWriteLoops                = 5
-	diskWriteMBsAvg               = 1000
-	diskWriteMBsPoor              = 500
-	availableMemoryMBsAvg         = 4000
-	availableMemoryMBsPoor        = 2000
-	totalMemoryMBsAvg             = 8000
-	totalMemoryMBsPoor            = 4000
+	diskOpsNumOfJobs      = 8
+	diskWriteSpeedMBsAvg  = 1000
+	diskWriteSpeedMBsPoor = 500
+	diskWriteIOPSAvg      = 200
```

Comment:
200 IOPS as ok/avg measure seems ill-advised, considering mechanical disks can reach 150 IOPS :)

My Linux desktop SSD reaches almost 80**k** IOPS in random 4KB block writes, more than 250**k** IOPS in the same class of reads.

[This](https://cloud.google.com/compute/docs/disks/benchmarking-pd-performance?hl=it) Google Cloud page has a good reference for `fio` command combos to test IOPS performance.

We should raise those numbers 10x at least IMO.

@OisinKyne thoughts?

Reply:
On my basic (but still latest) Macbook air with M3 chip I get ~1000 IOPS.

The numbers I have picked here are based only on my assumptions and I am definitely up for changing them if we can base them on some data!

Reply:
> I get ~1000 IOPS

I think what we're seeing here is an unoptimized IO path (`fio`'s default on macOS) versus an highly-optimized one on Linux (`libaio`): the latter resembles more what a real-world disk can do!

Reply:
Might very much be! I had the idea in my head to include the OS in this test as well, but didn't find big enough reason to do it on this first iteration. Now probably there is :)